_posts/2024-06-18-trino-fest-recap.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "Trino Fest 2024 recap"
+author: "Manfred Moser, Cole Bowden, Monica Miller"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2024/recap-banner.png
+---
+
+tbd
+<!--more-->
```

Comment:
Trino Fest 2024 is successfully in the books! While over 100 enthusiastic
members of the community gathered in Boston, over 650 virtual attendees 
joined us worldwide to learn from our expert speakers as they discussed
topics such as table formats, enhancements and optimizations, and use
cases with Trino both large and small.